src/components/chip-set/chip-set.scss

```diff
@@ -22,6 +24,15 @@ $mdc-text-field-fullwidth-bottom-line-color: $lime-text-field-bottom-line-color;
  * @prop --input-chip-set-selected-chip-color: Color of the highlight around selected chips in input chip-sets.
  */
 
+.mdc-chip {
+    background-color: $mdc-chip-background-color;
+    @include is-elevated-clickable;
+
+    &:hover {
+        background-color: $mdc-chip-background-color;
```

Comment:
Commits that change how components look shouldn't be of type `refactor` :)

(I'll fix.)